views/index.html.twig

```diff
@@ -81,3 +81,20 @@
         </div>
     </div>
 {% endblock %}
+
+{% block pagejs %}
+<script>
+$(document).ready(function () {
+  // Add badge for count of Deceased Participant Reports for review
+  var deceasedReportCount = 0;
+  $.ajax({
+    url: '/s/deceased-reports/stats'
```

Comment:
Hard-coding this for now. I use [FOSJsRoutingBundle](https://symfony.com/doc/master/bundles/FOSJsRoutingBundle/usage.html) in other projects, but this page has not been migrated over.

Reply:
Currently, the way we are passing routes to JS is through the `PMI.path` global object (see `base-head.html.twig` in both the Symfony and Silex templates). It's not necessarily the best solution, but just thought I'd point that out in case you think it makes sense to use.

Reply:
Thanks. If we started building it into the the overall "chrome" of the app (global alerts) I'd feel better about putting it there. For now, I think I'll leave it as the hard-coded URL since it is referenced from only the one page.